src/main/java/com/puppycrawl/tools/checkstyle/checks/metrics/BooleanExpressionComplexityCheck.java

```diff
@@ -353,7 +353,6 @@ private final class Context {
          */
         private Context(boolean checking) {
             this.checking = checking;
-            count = 0;
```

Comment:
Similar to a previous one, it is assigned 0 even if we don't do anything so this assignment is unnecessary.